packages/cmf/src/sagas/index.md

```diff
@@ -68,32 +68,33 @@ const options = {
 
 const { data, response } = yield call(http.get, `${API['dataset-sample']}/${datasetId}`, config, options);
 ```
-* The config object allow you to customize your http request
+
+#### Config
+
+The config object allow you to customize your http request
  + ```headers```, ```credentials```, ```method```, ```body``` will be merged recursively against other provided arguments and override those values.
  + ```security``` will be resolved and then merged
 
-* The options object allow you to configure cmf behavior.
+#### Options
+
+The options object allow you to configure cmf behavior.
 
   + The ```silent``` property to ```true``` avoid that cmf dispatch an action of type ```@@HTTP/ERRORS```.<br/>
   It could be usefull if you want to treat the request error on a specific way only and deal with it within your own saga.
 
-#### Advanced silent options
+  + The other properties are passed in the dispatched error action. You can pass whatever option you want, to pass them to you app error handler.
+
+Example
 
 ```javascript
 const options = {
-	silent: {
-		notify: false,
-		redirect: false,
-	},
+    notify: false,
```

Comment:
please add documentation about `silent : false` as it belongs to cmf

Plus providing aside properties will lead to the possibility to pass inconsistent options: eg: 
`{
silent: true,
notify: true,
redirect: true
}` which was avoided by the previous implementation and aligned documention

Reply:
This is an example. not a suggestion of what to write in your app.
It'll change for `toto`, `tata` to avoid confusion